week-1/2-mandatory/3-function-output.js

```diff
@@ -1,14 +1,23 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+/* This is a random numbers generating function that give a random number between 
+ 0 and 9 (both included). */
```

Comment:
Pretty much there! It's possible to generate a random number less than 10 here but greater than 9.